src/answers-umd.js

```diff
@@ -241,20 +237,32 @@ class Answers {
 
     this._onReady = parsedConfig.onReady || function () {};
 
+    const callPonyfillCssVariables = (callback) => {
+      if (!parsedConfig.disableCssVariablesPonyfill) {
+        this.ponyfillCssVariables({
+          onFinally: () => {
+            callback();
+          }
+        });
+      } else {
+        callback();
+      }
+    };
+
     if (parsedConfig.useTemplates === false || parsedConfig.templateBundle) {
       if (parsedConfig.templateBundle) {
         this.renderer.init(parsedConfig.templateBundle);
       }
 
-      this._onReady();
+      callPonyfillCssVariables(this._onReady.bind(this));
```

Comment:
I am worried about having our onReady be guarded by a successful call to `onFinally` by a third part library. I think the tradeoff here is

(1) call onReady whenever
* css variables may not be resolved as components show on the page, showing unstyled components until they are resolved (this would only happen on legacy browsers; IE11 afaik)
* onReady guaranteed to happen

(2) call onReady only when ponyfill is finished
* more risk in relying on third party lib to always call onFinally in every case
* longer wait time for components to show on the page on IE11

I don't swing either way, just expressing my worries about guarding our onReady because it could stop an experience from loading altogether

Reply:
I'm not too concerned about relying on a 3rd party library to invoke onFinally. We've thoroughly tested this item and their repository is well documented and maintained. I think we've done our due diligence. I think something would need to go very wrong, structurally, with their library for onFinally to not be invoked and that would have been caught by our testing. The longer wait time is another issue. Have you noticed significant slowness in IE11 with this change?